master/internal/resourcemanagers/kubernetes/pods.go

```diff
@@ -80,6 +80,15 @@ type pods struct {
 	configMapInterface typedV1.ConfigMapInterface
 }
 
+// PodsInfo contains information for pods.
+type PodsInfo struct {
+	NumAgents int
+	NumSlots  int
+}
+
+// ResourceSummarize summerize pods resource.
+type ResourceSummarize struct{}
```

Comment:
nit, to match other messages
```suggestion
type SummarizeResources struct{}
```